Makefile

```diff
@@ -19,7 +19,40 @@ COMMON_ARGS += --build-arg=https_proxy=$(https_proxy)
 empty :=
 space = $(empty) $(empty)
 
-TARGETS =  ca-certificates  cni  containerd cryptsetup dosfstools  eudev  fhs flannel-cni grub libinih liburcu ipmitool  iptables ipxe kernel  kmod  libaio libjson-c liblzma libpopt libressl  libseccomp  linux-firmware lvm2  musl  open-iscsi  open-isns raspberrypi-firmware runc  socat  syslinux u-boot  util-linux  xfsprogs
+TARGETS = \
+	ca-certificates \
```

Comment:
I don't see `openssl` in the list, should it be there?

Also It feels like this list was sorted at some point. Maybe we should make it sorted again as it's in this more convenient form for that right now?

Would be easier to spot any missing things there if it's sorted.

Reply:
> I don't see openssl in the list, should it be there?

Hmm, or it is part of `ipmitool`? :)

Reply:
yep, I think I missed `openssl`, and the list is (mostly) sorted, I will make sure it is sorted properly